pkg/log/zap/zap.go

```diff
@@ -154,9 +180,8 @@ func (o *Options) addDefaults() {
 	}
 
 	if o.Development {
-		if o.Encoder == nil {
-			encCfg := zap.NewDevelopmentEncoderConfig()
-			o.Encoder = zapcore.NewConsoleEncoder(encCfg)
+		if o.NewEncoder == nil {
+			o.NewEncoder = newConsoleEncoder
```

Comment:
Totally minor/non-blocking nit:

Move the `newConsoleEncoder` and `newJSONEncoder` functions into this file now that we're using them outside the context of flag parsing.